lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer_test.go

```diff
@@ -849,46 +848,26 @@ func assertEqualSubscriberData(t *testing.T, expectedProtos []*lte_protos.Subscr
 	}
 }
 
-func initializeDigestStore(t *testing.T) subscriberdb_storage.DigestStore {
-	db, err := sqorc.Open("sqlite3", ":memory:")
-	assert.NoError(t, err)
-	store := subscriberdb_storage.NewDigestStore(db, sqorc.GetSqlBuilder())
-	assert.NoError(t, store.Initialize())
-	return store
+func assertEqualAnyData(t *testing.T, expected []*any.Any, got []*any.Any) {
```

Comment:
I think you might be able to, either as a replacement for this fn, or within this fn, use https://github.com/gogo/protobuf/blob/master/proto/equal.go#L72

Reply:
I have tried using `proto.Equal`, unfortunately it doesn't seem to work with the `any.Any` type @hcgatewood

Reply:
Hm must be because we're using an outdated version. Will you add a comment then about why you're not using proto.Equal